test_psa_target.py

```diff
@@ -280,7 +279,7 @@ def _get_parser():
         "-t",
         "--toolchain",
         help="Build for the given toolchain (GNUARM)",
-        default="GNUARM",
+        default=None,
```

Comment:
Any specific reason why you removed the default? If so, could you update the `help` line above which implies that the default is GNUARM

Reply:
This is removed to give `-l` option run, if given it will force the script to run, it is on sync with - https://github.com/ARMmbed/mbed-os-tf-m-regression-tests/blob/master/build_tfm.py#L544